rest/server_context.go

```diff
@@ -946,6 +954,92 @@ func (sc *ServerContext) getDbConfigFromServer(dbName string) (*DbConfig, error)
 	return &config, nil
 }
 
+//////// STATS LOGGING
+
+func (sc *ServerContext) startStatsLogger() {
+
+	statsLogFrequencySecs := 5
+	if sc.config.Unsupported != nil && sc.config.Unsupported.Logging != nil {
+		statsLogFrequencySecs = sc.config.Unsupported.Logging.StatsLogFrequencySecs
```

Comment:
Should we set an absolute minimum that can't be overridden, even by the unsupported property?  If not, what does frequency=0 mean?

Reply:
Changed to check for `0`, which will silently revert back to default val.  Any other val will be considered valid.